server.js

```diff
@@ -0,0 +1,278 @@
+const net = require('net');
+const routes = [
```

Comment:
Rather than having a complex object like this. It's best to be able to build out the response as you need. This also allows for flexibility in extending this in the future.